packages.json

```diff
@@ -9295,7 +9295,7 @@
       "pqcrypto",
       "signing"
     ],
-    "description": "SPHINCS⁺ stateless hash-based signature scheme",
+    "description": "SPHINCS\u207A stateless hash-based signature scheme",
```

Comment:
I think unicode crap happened?

Reply:
Not @owlinux1000's fault and it doesn't matter too much IMO.